docs/extensions/basics/web-assets.md

```diff
@@ -110,8 +110,8 @@ In the above example:
   * `setFileName()` sets the file name
   * `setLate(true)` tells the asset injector to insert the `<link rel="stylesheet" href="path/web/koala.css">`
     at the end of the HTML `<body>`
-  * `setPriority(99)` tells the injector to insert this `<link rel="stylesheet" href="path/web/koala.css">`
-    *after* any other stylesheet tags in that location with a priority lower than 99
+  * `setPriority(5)` tells the injector when to insert the stylesheet. Lower
```

Comment:
Hmm... It's the opposite in core? Do you understand the previous text to imply that too? (I'm asking, since in my mind the previous text also said "lower first", just in a move convoluted way.)